specificatie/genereervariant/openapi.json

```diff
@@ -576,6 +576,19 @@
         "type" : "string"
       },
       "Reisdocument" : {
+        "allOf" : [ {
+          "$ref" : "#/components/schemas/ReisdocumentBasis"
+        }, {
+          "type" : "object",
+          "properties" : {
+            "inOnderzoek" : {
+              "$ref" : "#/components/schemas/ReisdocumentInOnderzoek"
+            }
+          },
+          "description" : "Een document dat nodig is voor reizen naar het buitenland.\n\nindicatieGeldig geeft aan of het reisdocument op dit moment nog geldig en in het bezit van de persoon is. Wordt gevuld op basis van de waarden die voorkomen in de tabel 'Geldig_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.\n"
```

Comment:
die API is er niet meer, Haal-Centraal-BRP-tabellen-bevragen API herformuleren? Landelijke tabellen?

Reply:
goed punt. dit is geen landelijke tabel, maar een door onszelf verzonnen "tabel" op basis van mogelijke waarden in het logisch ontwerp BRP. Deze lijst staat nu [tabelwaarden.csv](https://github.com/BRP-API/Haal-Centraal-BRP-tabellen-bevragen/blob/master/docs/tabelwaarden.csv) in de tabellen API repository. Wellicht moeten we dat document ook opsplitsen en verplaatsen naar de verschillende API repositories (personen bevragen, reisdocumenten, bewoning en historie). Of heb je een ander idee hoe we dit moeten vormgeven?

Reply:
Deze en volgende verwijzingen naar de tabellen API verwijderd.
- omschrijvingen voor veld met waarde uit landelijke tabel zijn gewijzigd naar "wordt gevuld met waarden uit de landelijke tabel '{{naam van de tabel}}'."
- omschrijvingen voor veld met waarde uit eigen tabellen zijn gewijzigd naar "wordt gevuld met waarden voor '{{naam van de tabel}}' in 'tabelwaarden.csv'."